src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap';
```

Comment:
Using '@import' for Google Fonts in SCSS is not recommended as it can lead to performance issues. It's better to include the font via a <link> tag in the HTML file's <head> section.